app/main.py

```diff
@@ -1,25 +1,87 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self._name = ""
+
+    def __set_name__(self, owner: SlideLimitationValidator, name: str) -> None:
+        self._name = "_" + name
+
+    def __get__(
+            self,
+            obj: SlideLimitationValidator,
+            object_type: type | None = None
+    ) -> int:
```

Comment:
```suggestion
    def __get__(
            self,
            instance: SlideLimitationValidator,
            owner: ...
    ) -> int:
```
+
specify owner typehint